packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -215,6 +215,11 @@ export const ECS_SERVICE_EXTENSIONS_ENABLE_DEFAULT_LOG_DRIVER = '@aws-cdk-contai
  */
 export const EC2_UNIQUE_IMDSV2_LAUNCH_TEMPLATE_NAME = '@aws-cdk/aws-ec2:uniqueImdsv2TemplateName';
 
+/**
+ * Enable this feature flag to have Events emit a warning upon synthesis or deployment when `minute` is not defined in CronOptions.
+ */
+export const EVENTS_WARNING_CRON_MINUTES_NOT_SET = '@aws-cdk/aws-events:cron';
```

Comment:
Do you think we need a flag to enable the warning? It wouldn't really change behavior, so it feels unnecessarily safe.

I was thinking we would add a feature flag to *actually* change behavior: 

I think we can agree that the behavior should have been: an undefined `minute` should evaluate to `0`, not to `*` (Can we? Maybe not?). We can make that the NEW default behavior, and put that default behavior behind a feature flag.

I'm also fine not doing that at all and sticking to a warning though.